Umbraco-Cloud/Troubleshooting/Log-files/index.md

```diff
@@ -36,6 +36,22 @@ When you're in Kudu, you can go up to your `site` folder as described in the 5 s
 
 - `log.log` shows you the same output you will have seen when pushing your changes using Git, it will show you what happened during the push and if any errors occurred. This file is especially useful when trying to find errors for deploys using the portal (so from dev > live or from dev > staging > live). Even though the last line may end with "Deployment successful" it is possible that there were errors or suspicious messages before that so make sure to give them a read.
 
+## Cleaning up the environment logs viewer
+
+On Cloud environments all errors are logged to a database table, which is what is seen in the portal under each environment. If you leave too many unread log messages it can cause timeouts when you go to see your errors. 
+
+Since these errors are stored in your database you can go clean them up by [accessing the environment database](../../Databases/Cloud-Database).
+
+The only way to resolve the issue is to open the database for the environment you want to delete logs from by using this guide - https://our.umbraco.com/Documentation/Umbraco-Cloud/Databases/Cloud-Database/
```

Comment:
```suggestion
Since the errors are stored in your database it is possible to clean them up. To do this, start by [accessing the database](../../Databases/Cloud-Database) for the environment where you want to run the cleanup.
```